setup.py

```diff
@@ -44,7 +44,7 @@
     console_scripts.append('msidp = mslib.msidp.idp:main')
 
 setup(
-    name="mss",
+    name="mslib",
```

Comment:
This name change was necessary because of the name collision with the mss package on PyPI, which we also depend on. Otherwise the editable install of this MSS into the pixi environment was impossible.

Reply:
I am not clear about the consequences. Does this mean that we would have to call ourselves MSLIB on conda-forge and within the project? I would then need to initiate a migration; it also has implications for publications.

If it all comes down to a renaming everywhere, then open-mss fits well with gh and open-mss.org.

The other mss project is called python-mss on conda-forge. If we choose a new name, we should do so carefully because "Mission Support System" is a very generic term.

D.h. am nächsten vom mapping her, wäre da open-mss.

Reply:
This is only the name of the python package defined by this setup.py file. It does not mean that we have to change the package name anywhere else AFAIU. PyPI should be the only package registry that actually enforces matching package names in the python package and the project on the index, if it even does that.

I've called it mslib because that is the name of the main module inside, but I agree it isn't a nice name. open-mss would sound good to me. `python-mss` is called that on conda-forge because all pure-python packages should get a `python-` prefix there AFAIK.

Reply:
let us change that name already to open-mss. Then we have used that already there and at some point someone again tries to create a pypi package it needs not another renaming.

We can change the inner later anyway.

Reply:
I changed it. We should reserve the open-mss name on PyPI though, so we don't run into this again in the future.

Reply:
yes, I do

Reply:
https://test.pypi.org/project/open-mss/

just learned a global installed pixi has precedence

```
(venv) (base) reimarbauer@MacBook-Pro-von-Reimar venv % bin/mss
open-mss is a conda-forge package.You can install it with pixi.
Get pixi from https://pixi.sh/latest/ for your operation system.

pixi global install mss

Usage:
    msui -h
    mswms -h
    mscolab -h

```
[our pypi project
](https://github.com/Open-MSS/open-mss)